theano/tensor/blas.py

```diff
@@ -160,6 +160,31 @@
 # We need to define blas.ldflag before we try to import scipy.
 # Otherwise, we give an optimization warning for no reason in some cases.
 def default_blas_ldflags():
+    """This is a generator. It work in 2 step. The first we guess a
+    default blas, then we test it. If it fail, we return an empty
+    blas.
+
+    This is needed for anaconda on Windows. I wasn't able to find how
+    to detect if the mkl from anaconda can be reused or not. I was not
+    able to find a way to test it with try_flags correctly. Also, this
+    will test the real code, so we don<t need to update the test in
+    case the software change. This also enable the test for all
+    case.
```

Comment:
Done.

On Tue, Apr 14, 2015 at 5:52 PM, Pascal Lamblin <notifications@github.com>
wrote:

> In theano/tensor/blas.py
> <https://github.com/Theano/Theano/pull/2764#discussion_r28374630>:
>
> > @@ -160,6 +160,31 @@
> >  # We need to define blas.ldflag before we try to import scipy.
> >  # Otherwise, we give an optimization warning for no reason in some cases.
> >  def default_blas_ldflags():
> > +    """This is a generator. It work in 2 step. The first we guess a
> > +    default blas, then we test it. If it fail, we return an empty
> > +    blas.
> > +
> > +    This is needed for anaconda on Windows. I wasn't able to find how
> > +    to detect if the mkl from anaconda can be reused or not. I was not
> > +    able to find a way to test it with try_flags correctly. Also, this
> > +    will test the real code, so we don<t need to update the test in
> > +    case the software change. This also enable the test for all
> > +    case.
>
> "cases"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2764/files#r28374630>.
>